Xamarin.Forms.Platform.UAP/FormsTextBox.cs

```diff
@@ -167,6 +172,30 @@ protected override void OnApplyTemplate()
 				if (_DeleteButtonVisibleStateGroups != null)
 					_DeleteButtonVisibleState = _DeleteButtonVisibleStateGroups.States.SingleOrDefault(s => s.Name == "ButtonVisible");
 			}
+
+			_scrollViewer= (Windows.UI.Xaml.Controls.ScrollViewer)GetTemplateChild("ContentElement");
```

Comment:
us `as` for casting, to prevent exception if the template is modified

Reply:
I'll submit a PR to fix that and the other casts in OnApplyTemplate.  I also noticed that we no longer need to grab a reference to the border element, 1 because it's no longer reference except to set it and phone is no longer supported on UWP

Reply:
#11285 Submitted